pandas/core/frame.py

```diff
@@ -6990,8 +6982,7 @@ def applymap(self, func) -> "DataFrame":
         0  3  4
         1  5  5
 
-        Note that a vectorized version of `func` often exists, which will
-        be much faster. You could square each number elementwise.
+        You could square each number elementwise.
```

Comment:
This part doesn't need changing, the vectorised version is still faster:
```
In [28]: s = pd.Series(np.random.randn(10000))                                                                                                               

In [29]: %timeit s.apply(lambda x: x**2)                                                                                                                     
6.39 ms ± 5.73 µs per loop (mean ± std. dev. of 7 runs, 100 loops each)

In [30]: %timeit s**2                                                                                                                                        
243 µs ± 4.38 µs per loop (mean ± std. dev. of 7 runs, 1000 loops each)

In [31]: pd.testing.assert_series_equal(s.apply(lambda x: x**2), s**2) 
```

Reply:
Oh okay. Thanks a lot. Included it.